extensions/meme_rater.py

```diff
@@ -62,34 +88,57 @@ async def main(event: hikari.GuildMessageCreateEvent) -> None:
             continue
         image_url = attachment.url
         try:
-            rating = await get_meme_rating(image_url, event.author.username, att_ext)
-            ratings.append(int(rating))
+            rating = await get_meme_rating(image_url, event.author.username)
+            ratings.append(rating)
         except Exception as e:
             continue
+    
+    await rate_meme(event.message, ratings)
+
+
+async def rate_meme(message: hikari.Message, ratings: list[int]) -> None:
```

Comment:
So what happens if this function runs twice at the same time because the create & update events fire near each other?

Reply:
I'm hoping the database will magically handle this